From 69a92f4576a1e789ba2fcf957164d2c4013020c5 Mon Sep 17 00:00:00 2001
From: Otavio Salvador <otavio@ossystems.com.br>
Date: Wed, 2 Dec 2015 13:36:25 +0000
Subject: [PATCH] Stop using Git to write local version
Organization: O.S. Systems Software LTDA.

The standard version does not use a Git repository so we should not
use Git to identify the commit of the build as it can end getting the
version from a wrong repository and can be misleading.

Upstream-Status: Pending

Signed-off-by: Otavio Salvador <otavio@ossystems.com.br>
[Downloaded from
https://github.com/Freescale/meta-fsl-arm/blob/master/recipes-graphics/xorg-driver/xf86-video-imxfb-vivante/Stop-using-Git-to-write-local-version.patch]
Signed-off-by: Andrew Webster <awebster@arcx.com>
---
 EXA/src/makefile.tc         | 6 +++---
 FslExt/src/makefile.tc      | 6 +++---
 util/autohdmi/makefile.tc   | 6 +++---
 util/pandisplay/makefile.tc | 6 +++---
 4 files changed, 12 insertions(+), 12 deletions(-)

diff --git a/EXA/src/makefile.tc b/EXA/src/makefile.tc
index 0b9a9e6..ec6e68d 100644
--- a/EXA/src/makefile.tc
+++ b/EXA/src/makefile.tc
@@ -52,8 +52,8 @@ prefix ?= /usr
 sysroot ?= /
 
 # get git commit number
-COMMITNR := `git log -n 1 --format=%H`
-DIRTY := `git diff-index --quiet HEAD || echo '-dirty'`
-LOCAL_CFLAGS += -DCOMMIT="${COMMITNR}${DIRTY}"
+#COMMITNR := `git log -n 1 --format=%H`
+#DIRTY := `git diff-index --quiet HEAD || echo '-dirty'`
+#LOCAL_CFLAGS += -DCOMMIT="${COMMITNR}${DIRTY}"
 
 
diff --git a/FslExt/src/makefile.tc b/FslExt/src/makefile.tc
index 0b9a9e6..ec6e68d 100644
--- a/FslExt/src/makefile.tc
+++ b/FslExt/src/makefile.tc
@@ -52,8 +52,8 @@ prefix ?= /usr
 sysroot ?= /
 
 # get git commit number
-COMMITNR := `git log -n 1 --format=%H`
-DIRTY := `git diff-index --quiet HEAD || echo '-dirty'`
-LOCAL_CFLAGS += -DCOMMIT="${COMMITNR}${DIRTY}"
+#COMMITNR := `git log -n 1 --format=%H`
+#DIRTY := `git diff-index --quiet HEAD || echo '-dirty'`
+#LOCAL_CFLAGS += -DCOMMIT="${COMMITNR}${DIRTY}"
 
 
diff --git a/util/autohdmi/makefile.tc b/util/autohdmi/makefile.tc
index c9de0a6..d0a468c 100644
--- a/util/autohdmi/makefile.tc
+++ b/util/autohdmi/makefile.tc
@@ -64,8 +64,8 @@ prefix ?= /usr
 sysroot ?= /
 
 # get git commit number
-COMMITNR := `git log -n 1 --format=%H`
-DIRTY := `git diff-index --quiet HEAD || echo '-dirty'`
-LOCAL_CFLAGS += -DCOMMIT="${COMMITNR}${DIRTY}"
+#COMMITNR := `git log -n 1 --format=%H`
+#DIRTY := `git diff-index --quiet HEAD || echo '-dirty'`
+#LOCAL_CFLAGS += -DCOMMIT="${COMMITNR}${DIRTY}"
 
 
diff --git a/util/pandisplay/makefile.tc b/util/pandisplay/makefile.tc
index 28732b9..bf54c20 100644
--- a/util/pandisplay/makefile.tc
+++ b/util/pandisplay/makefile.tc
@@ -64,8 +64,8 @@ prefix ?= /usr
 sysroot ?= /
 
 # get git commit number
-COMMITNR := `git log -n 1 --format=%H`
-DIRTY := `git diff-index --quiet HEAD || echo '-dirty'`
-LOCAL_CFLAGS += -DCOMMIT="${COMMITNR}${DIRTY}"
+#COMMITNR := `git log -n 1 --format=%H`
+#DIRTY := `git diff-index --quiet HEAD || echo '-dirty'`
+#LOCAL_CFLAGS += -DCOMMIT="${COMMITNR}${DIRTY}"
 
 
-- 
2.1.4

